service-worker.js

```diff
@@ -12,7 +12,11 @@
  */
 
 importScripts(
+<<<<<<< HEAD
```

Comment:
@ath-chen looks like the merge conflict is still here

Reply:
updated